### DISCUSSION TITLE
How can I return the default behavior of ctrl+down in macOS?

### DISCUSSION THREAD

**Discussion by SteveCohen** [vslinko/obsidian-outliner#192] [Q&A] [answered]
In MacOS, command+down (and cmd+up) jumps to the bottom (top) of any text box. This also applies in Obsidian.  When Obsidian-Outliner is activated this command is overriden and instead unfolds (folds) an item. 

The readme confirms this is the intended "default" key; does "default" imply alternate hotkeys can be set?  Alternatively: Could an alternative option be provided?  Or could a switch be added to allow folding/unfolding hotkeys be disabled by the user?

Thank you for your work. Fantastic plugin!

**Comment by vslinko** [comment, marked as answer]
@SteveCohen Hello!

You can change the hotkeys in the Obsidian settings and restore the system behavior.
<img width="1459" alt="Screenshot 2021-09-04 at 16 06 28" src="https://user-images.githubusercontent.com/205823/132095397-46b51f3f-4312-4b26-9015-70dad61ac226.png">

**Reply by eryajf** [reply]
> @SteveCohen你好！
> 
> 您可以更改 Obsidian 设置中的热键并恢复系统行为。 <img alt="截图 2021-09-04 at 16 06 28" width="1459" src="https://user-images.githubusercontent.com/205823/132095397-46b51f3f-4312-4b26-9015-70dad61ac226.png">

Thank you, this has solved my problem.

**Comment by cweekly** [comment]
Closely-related issue: https://github.com/vslinko/obsidian-outliner/issues/200
